spec.bs

```diff
@@ -1669,8 +1669,12 @@ and a [=real time reporting contributions map=] |realTimeContributionsMap|:
   1. Let |topLevelDirectFromSellerSignalsRetrieved| be false.
   1. [=list/For each=] |component| in |auctionConfig|'s [=auction config/component auctions=],
     [=parallel queue/enqueue steps|enqueue the following steps=] to |queue|:
-    1. Let |compWinnerInfo| be the result of running [=generate and score bids=] with |component|,
-      |auctionConfig|, |global|, |bidIgs|, |bidDebugReportInfoList|, and |realTimeContributionsMap|.
+    1. If |component|'s [=auction config/server response=] is not null:
+      1. Let |compWinnerInfo| be the result of running [=parse and validate server response=] with |component|,

```

Comment:
do we return failure at line 1678 to terminate the auction in this case? Or do we only ignore this single component auction? https://crsrc.org/c/content/browser/interest_group/interest_group_auction.cc;drc=a150b50c0ff706af12c449c7fccd3cf2745e2061;l=3032